### PR TITLE
added tags -> productivity/_index.md, vscode/_index.md, crypto/_index…

### PR DESCRIPTION
**Pull request by asifthewebguy**
….md, javascript/_index.md, css/_index.md, tailwind/_index.md, react-native/_index.md, d3/_index.md, animation/_index.md, go/_index.md, svg/_index.md, hooks/_index.md, intl/_index.md, database/_index.md, threejs/_index.md, payments/_index.md, webrtc/_index.md, authentication/_index.md, nginx/_index.md, rpi/_index.md, pwa/_index.md, ssr/_index.md, security/_index.md, wasm/_index.md, fauna/_index.md, sql/_index.md, meilisearch/_index.md, quickstart/_index.md, testing/_index.md, auth/_index.md, typescript/_index.md, deno/_index.md, regex/_index.md, gcp/_index.md, forms/_index.md, graphql/_index.md, electron/_index.md, svelte/_index.md, ios/_index.md, aws/_index.md, compsci/_index.md, slack/_index.md, ui/_index.md, data-modeling/_index.md, deckgl/_index.md, google-maps/_index.md, apollo/_index.md, analytics/_index.md, algolia/_index.md, minimum-viable-product/_index.md, cloud-run/_index.md, nest/_index.md, dialogflow/_index.md, ionic/_index.md, puppeteer/_index.md, sendgrid/_index.md, paypal/_index.md, push-notifications/_index.md, advanced/_index.md, performance/_index.md, pubsub/_index.md, linux/_index.md, flare2d/_index.md, big-query/_index.md, machine-learning/_index.md, python/_index.md, rxdart/_index.md, twa/_index.md, cloud-storage/_index.md, realtimedb/_index.md, jest/_index.md, ml/_index.md, google-calendar/_index.md, i18n/_index.md, web-components/_index.md, tensorflow/_index.md, seo/_index.md, ngrx/_index.md, fcm/_index.md, mapbox/_index.md, nosql/_index.md

### CODE REVIEW

**Comment by asifthewebguy** [comment]
removed my version of python tag, contains almost all the missing tags

**Comment by etzl** [comment]
@asifthewebguy I think [this commit](https://github.com/fireship-io/fireship.io/pull/557/commits/26441778b6f2e7453d205b9f0b9e85462e6cc51d) is causing a merge conflict. If that's the case you can revert back with 

`git reset --soft <SHA>`  to keep your current changes then commit again, and force push.

File: hugo/content/tags/crypto/_index.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+title: Crypto
+description: the art of writing or solving codes.
```
Comment (etzl) [inline hugo/content/tags/crypto/_index.md:3]: I don't think that's the definition

File: hugo/content/tags/css/_index.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+title: CSS
+description: CSS stands for Cascading Style Sheets· CSS is the language we use to style a Web page.
+link:
```
Comment (etzl) [inline hugo/content/tags/css/_index.md:4]: https://www.w3.org/TR/CSS/#css might be a good link. Wikipedia uses it. If I were you I'd ask

File: hugo/content/tags/css/_index.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+---
+title: CSS
+description: CSS stands for Cascading Style Sheets· CSS is the language we use to style a Web page.
```
Comment (etzl) [inline hugo/content/tags/css/_index.md:3]: I'm not a fan of your description, But I might be biased

**Comment by etzl** [comment]
#25

**Comment by asifthewebguy** [comment]
> @asifthewebguy I think [this commit](https://github.com/fireship-io/fireship.io/pull/557/commits/26441778b6f2e7453d205b9f0b9e85462e6cc51d) is causing a merge conflict. If that's the case you can revert back with
> 
> `git reset --soft <SHA>` to keep your current changes then commit again, and force push.

done

File: hugo/content/tags/css/_index.md (hunk 2 above)
Comment (asifthewebguy) [reply hugo/content/tags/css/_index.md]: added the link

File: hugo/content/tags/css/_index.md (hunk 3 above)
Comment (asifthewebguy) [reply hugo/content/tags/css/_index.md]: updated the description 😊

**Comment by etzl** [comment]
@asifthewebguy Great! I hope someone will approve your work sooner if it doesn't need modification

**Pull request closed without merging**